omero/features/share/index.html

```diff
@@ -0,0 +1,37 @@
+---
+layout: omero-features
+title: OMERO Features
+feature: Share
+usergroup: omero
+redirect_from:
+  - ../omero/share/
```

Comment:
I think this redirect related to the broken link reported by the linkcheck in the CI build. Since this is a new page, is there a need to introduce this redirect? Previous ones were primarily introduced because we moved the URLs.

Reply:
no real need, I just went for "replace same with same" tactics - looking for the word "organize" everywhere and replacing it with the new word "share" (I copied the template of the new page from the "organize" page

Reply:
fixed in https://github.com/ome/www.openmicroscopy.org/pull/447/commits/2db09f5b56dbd1f29611e5da7c3e7876f17fc3b7